sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -77,7 +77,7 @@ extensions:
       title: "copy openshift images from docker storage to CRI-O storage"
       timeout: 10800
       script: |-
-        for i in $(docker images --format '{{.Repository}}:{{.Tag}}' | grep "openshift\/origin"); do
+        for i in $(docker images --format '{{.Repository}}:{{.Tag}}' | grep -v "<none>" | grep -v "latest" | grep "openshift\/origin"); do
```

Comment:
Are you sure you don't want `latest`?

Reply:
yeah, afaict origin tests just use the built origin release which always use specific commits. https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_conformance_crio/144/ just went fine